### PR TITLE
Exercise description update

### PR DESCRIPTION
**Pull request by alex4200**

### CODE REVIEW

**Comment by alex4200** [comment]
@antonelepfl I think the changes in text for th exercises are complete. I appreciate if you also can check the changes quickly. Thanks.

File: notebooks/week_3/bouton_density.ipynb [hunk 1]
```diff
@@ -260,12 +260,12 @@
    "metadata": {},
    "source": [
     "### Exercise #1\n",
-    "Calculate the average bouton density in an interneuron and a neuron in general. Store the answer in _ans_1a_ and _ans_1b_.\n",
+    "Calculate the average bouton density in an interneuron and a neuron (either excitatory or inhibitory). Store the mean value and the standard deviation in a list each for the interneuron (answer _ans_1a_) and the neuron (answer_ans_1b_).\n",
```
Comment (antonelepfl) [inline notebooks/week_3/bouton_density.ipynb]: > ... list each for the interneuron
Not very clear this part. In the same list?

File: notebooks/week_3/bouton_density.ipynb [hunk 2]
```diff
@@ -260,12 +260,12 @@
...
     "\n",
-    "__Hint__: to query GIDs of interneurons use Cell.SYNAPSE_CLASS. When you query all the interneurons, you do not need any filters.\n",
+    "Hint: to query GIDs of interneurons use Cell.SYNAPSE_CLASS. When you query all the interneurons, you do not need any filters.\n",
     "\n",
     "### Exercise #2\n",
-    "Calculate bouton density not in the central column and store the answer in _ans_2a_ and _ans_2b_. Is it lower, bigger or the same of the bouton density calculated in the central column? Why?"
+    "Calculate the average bouton density in a neuron (either excitatory or inhibitory) in column 0 (zero). Similarly to the previous exercise, store mean and standard deviation in a list and assign it to _ans_2a_. Are the values lower, bigger or the same of the bouton densities calculated in the central column? Store 'lower', 'bigger', or 'same' in _ans_2b_. As for ungraded exercise, can you explain why you found or not found a difference?"
```
Comment (antonelepfl) [inline notebooks/week_3/bouton_density.ipynb]: > in column 0
What this column 0 defined? How do they know which target they have to use?

File: notebooks/week_3/nsyns_per_conn.ipynb [hunk 3]
```diff
@@ -274,10 +274,10 @@
    "metadata": {},
    "source": [
     "### Exercise #1\n",
-    "Calculate the average number of synapses per connection in the four classes of connections (EE, EI, IE, II). Put the answer in a list called _ans_1_. Respect the order (EE, EI, IE, II).\n",
+    "Calculate the average number of synapses per connection in the four classes of connections (EE, EI, IE, II). Put the answers in a list called _ans_1_ in the order (EE, EI, IE, II).\n",
     "\n",
     "### Exercise #2\n",
-    "Calculate the distribution of number of synapses from SP_PVBC to SP_PC. Put the answer in a list named _ans_2_ with mean and std"
+    "Calculate the distribution of number of synapses from SP_PVBC to SP_PC. Provide the (mean, std) as a 2-tuple for answer _ans_2_."
```
Comment (antonelepfl) [inline notebooks/week_3/nsyns_per_conn.ipynb]: > 2-tuple
?

File: notebooks/week_3/bouton_density.ipynb (hunk 1 above)
Comment (alex4200) [reply notebooks/week_3/bouton_density.ipynb]: ans_1a: One list with two elements for the "average bouton density in an interneuron". First element is the mean, second element is the standard deviation. 
ans_1b: One list with two elements for the "average bouton density in a neuron". First element is the mean, second element is the standard deviation. 

Not sure how to phrase it shorter.

File: notebooks/week_3/bouton_density.ipynb (hunk 2 above)
Comment (alex4200) [reply notebooks/week_3/bouton_density.ipynb]: Column 0 is a column in the brain part?

File: notebooks/week_3/nsyns_per_conn.ipynb (hunk 3 above)
Comment (alex4200) [reply notebooks/week_3/nsyns_per_conn.ipynb]: `Provide a list for _ans_2_, with the first element being the mean value and the second element being the standard deviation of the number of synapses from SP_PVBC to SP_PC.`

Better?

**Comment by alex4200** [comment]
I pushed some changes replying to your suggestions.

**Comment by antonelepfl** [comment]
Yeah much better now Alex! Thank you

The remaining thing is to replace `[Insert instruction on how to do the peer review]` by the actual instructions

**Comment by alex4200** [comment]
@romani79 Can you please provide the peer review instructions, so we can merge the text updates?

**Comment by alex4200** [comment]
According to Armando, these placeholder can be removed. I did remove them. 

I guess we can merge these changes.

**Pull request merged**